### ISSUE TITLE
can we expose /metrics in django-rq?

### ISSUE THREAD

**Issue by laixintao** [rq/django-rq#503]
Hi, I want to use Prometheus to collect metrics of rq.

we have https://github.com/mdawar/rq-exporter already, but I don't like the idea that we need to a separate wsgi app(process) to achieve this, since we already have django.

Can we expose metrics from django-rq?

**Comment by selwin** [comment]
Definitely, can you open a PR for this? Probably a simple view displaying metrics for one of the selected queues.

**Comment by laixintao** [comment]
OK for queues I want to make it default to expose all queues but configurable via `settings`.

**Comment by selwin** [comment]
I think having this as a config is not very useful. Better to show all queues and allow users to select which queues they want to show and refresh the page.
